azure-migrate/migrate-at-scale-vmware-agentles/Agentless VMware automation/AzMigrate_StartReplication.ps1

```diff
@@ -203,6 +203,25 @@ Function ProcessItemImpl($processor, $csvItem, $reportItem) {
         }
     }
 
+    $testNetworkId = $csvItem.TEST_NETWORK_ID
```

Comment:
Take Vnet name instead of ID. You can check how it is implemented for TARGET_VNET_NAME.